mmv1/third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -552,17 +552,18 @@ func TestAccComputeDisk_encryptionKMS(t *testing.T) {
 		},
 	})
 }
-<% unless version = 'ga' -%>
+
+<% unless version == 'ga' -%>
 func TestAccComputeDisk_pdHyperDiskEnableConfidentialCompute(t *testing.T) {
 	t.Parallel()
 
-
 	context := map[string]interface{}{
 		"random_suffix":        RandString(t, 10),
-		"kms":                  BootstrapKMSKey(t),
+		"kms":                  BootstrapKMSKey(t).CryptoKey.Name,
```

Comment:
I've seen in other test file they bootstrap the cryptoKey by putting `acctest.BootstrapKMSKey(t).CryptoKey.Name`, perhaps that could fix the test?